application/settings.ts

```diff
@@ -2,31 +2,29 @@
 // returns based on state i.e. uses envName, workflowName, etc passed at init
 export function getSettings(envName: string, workflowName: string) {
 
+  const nfBucket = envName == "prod" ? "org.umccr.data.oncoanalyser" :  `umccr-temp-${envName}`;
```

Comment:
Will it be too much of changes, if this instead write to as follows. 

Since we are now pointing to new bucket for PROD; if not `prod` then use --
- `umccr-primary-data-dev` if DEV
- `umccr-primary-data-stg` if STG

i.e.

```
const nfBucket = envName == "prod" ? "org.umccr.data.oncoanalyser" : `umccr-primary-data-${envName}`;
```

If we can do that, then OncoAnalyser result will be straight away pop up in Portal (in this case, DEV Portal & Staging Portal respectively) -- without needing any update at Portal side.

Because these buckets are already monitored by Portal. 
So 0 work need on Portal side, win! 😺

See

https://github.com/umccr/infrastructure/blob/b399aee9f01245ed451cee170919c28d7cd971bf/terraform/stacks/umccr_data_portal/app/s3.tf#L4-L11

Reply:
@reisingerf comment, pls! 😬 Happy to take your lead, here.

Reply:
Having results appear in the data portal without extra changes would be nice. Does the `analysis_data` and `temp_data` prefix work with the `umccr-primary-data-{dev,stg}` buckets? And would we need to add lifecycle policies as we've done for the data account bucket?

Reply:
Yes, those prefixes should work. Ah. Good point, we might still need to attach life cycle, as well as to exclude `temp_data` from Portal monitor.

Okay, in that case, let us go ahead with `umccr-temp-${envName}` as-is for now. I will work on Portal side a tad later and, regroup discussion on this.

If need be, it is still moving from bucket A to bucket B. So, this should be okay.

Reply:
Hm... looks like you found a workable solution?

I don't mind using `umccr-temp-${envName}` buckets for the temporary data. It's not monitored by the portal, so should not need anything from that side. Two things to consider/check though:

- not sure those buckets have lifecycle policies attached yet and with workflows producing data there, that should be in place to avoid data piling up.
- I guess we are currently keeping the output prefixes static and only change the buckets depending on deployment account? In that case those prefixes may not be specific enough in the generic temp bucket(s)?

Reply:
Yup. We need to figure how to exclude S3 event notification by prefix filtering i.e. not to index any file drops to `temp_data` but others. This use case has not verified yet at Portal side. 

So, let keep them separate bucket in `umccr-temp-${envName}` for now. Not to block oncoanalyser progress. 

I will investigate Portal side soon. Once the S3 event prefix filtering mechanism in place, we can do the data migration from `umccr-temp-${envName}` to the new destination where Portal monitored the bucket. It will also be testing this setup at Portal by then.

Reply:
Yep, the `umccr-temp-dev` but not`umccr-temp-stg` bucket has a lifecycle policy.

The concern with the output prefixes from my side was that they might break convention in the `umccr-primary-data-{dev,stg}` bucket, though we won't be using those buckets at this point. For the development/staging accounts, I think the prefixes are okay to go straight into the temp buckets for now and then adjust once we have a better idea how to integrate into the portal.

Reply:
Hey Flo; we will merge ahead for now, k. Tq for stop by. You stay rest. We won't mess up, promise! 😬

Reply:
Sorry, didn't mean to stall anything. I thought I'd approved already.